src/Database/Database.php

```diff
@@ -1823,14 +1820,14 @@ public function escapeArray(&$arr, bool $add_quotation = false)
 	/**
 	 * Replaces a string in the provided fields of the provided table
 	 *
-	 * @param string $table_name Table name
+	 * @param string $table  Table name
```

Comment:
Please update the documentation to reflect the type change with the optional schema prefix.